<?xml version="1.0" encoding="utf-8"?>

<launch>

	<rosparam file="$(find grasp_learning)/param/media_varianza.yaml" command="load"/>
	
		<node name="media_varianza" pkg="grasp_learning" type="media_varianza" respawn="false" output="screen"/>

</launch>